aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -352,6 +354,28 @@ public class AwsProperties implements Serializable {
 
   public static final String HTTP_CLIENT_TYPE_DEFAULT = HTTP_CLIENT_TYPE_URLCONNECTION;
 
+  /**
+   * Used to configure the connection timeout in milliseconds for {@link
+   * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when {@link
+   * #HTTP_CLIENT_TYPE} is set to {@link #HTTP_CLIENT_TYPE_APACHE}
+   *
+   * <p>For more details, see
+   * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+   */
+  public static final String APACHE_HTTP_CLIENT_CONNECTION_TIMEOUT_MS =
```

Comment:
sorry I totally forget we us `http-client` as prefix, so this should probably be `http-client.apache.xxxx`